pkgs/development/tools/open-policy-agent/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  pname = "open-policy-agent";
+  version = "v0.9.2";
```

```suggestion
  version = "0.9.2";
```
Version [must start with a digit](https://nixos.org/nixpkgs/manual/#sec-package-naming).

---

Oops, I know it moreover :/